Add Slovak translations

romana-la opened the pull request:
## Summary

Translation of the documentation into Slovak language.

## Related Issues

Is part of #738


## Checklist

- [x] This pull request has a meaningful title and description.
- [x] I have read and followed the [Contributing guidelines](https://github.com/danielschuster-muc/potter-db/blob/master/CONTRIBUTING.md).
- [x] My code follows the conventions and coding style of this project.
- [x] All tests pass and I have added tests for my changes <!-- if applicable -->
- [x] I have updated necessary documentation <!-- if applicable -->

danielschuster-muc commented on docs/pages/index.sk.mdx:
```diff
@@ -0,0 +1,89 @@
+Či už ste developer vyvíjajúci aplikáciu týkajúcu sa Harryho Pottera alebo len zvedavý fanúšik, táto dokumentácia vás prevedie tajmi nášho API.
+
+## Čo je to Potter DB: API?
+Potter DB: API je REST a GraphQL rozhranie pre rozsiahly repozitár dát z magického sveta Harryho Pottera. My sa staráme o dáta, zatiaľ čo vy sa môžete sústrediť na vývoj vašej aplikácie. Táto API vám umožní:
+- získať detailné informácie o vašich obľúbených postavách (`characters`) vrátane informácií o ich pozadí a fakulte
+- získať ucelený prehľad zaklínadiel (`spells`) a odvarov (`potions`) z čarodejníckeho sveta
+- získať podrobné informácie o knihách (`books`) a filmoch (`movies`) zo sveta Harryho Pottera
```
It's nice that you added the english translation / key of the resources here.

danielschuster-muc commented on docs/pages/resources.sk.mdx:
```diff
@@ -0,0 +1,30 @@
+
+<Cards>
+  <Card
+    title="Knihy (books)"
```
I am not quite sure if the english translations are needed here.

danielschuster-muc replied:
I mean it's not bad to have, but what do you think, is it needed to understand it?

danielschuster-muc commented on docs/pages/resources/_meta.sk.json:
```diff
@@ -0,0 +1,8 @@
+{
+  "books": "Knihy (books)",
+  "chapters": "Kapitoly (chapters)",
+  "characters": "Postavy (characters)",
+  "movies": "Filmy (movies)",
+  "potions": "Odvary (potions)",
+  "spells": "Zaklínadlá (spells)"
+}
```
same here

danielschuster-muc commented on docs/pages/resources/chapters.sk.mdx:
```diff
@@ -0,0 +1,51 @@
+# Kapitoly (chapters)
```
also here and on the other pages, do we really need the english translation of the resource keys?

danielschuster-muc replied:
```suggestion
# Kapitoly (chapters)

```

danielschuster-muc commented on docs/pages/apis/graphql.sk.mdx:
```diff
@@ -0,0 +1,47 @@
+
+Na získanie a zmenu dát cez GraphQL API pošlite požiadavku POST koncovému bodu GraphQL (`/graphql`). Telo požiadavky musí obsahovať váš dopyt a premenné a musí byť vo formáte JSON.
+
+## Dopytovanie dát
+Vďaka GraphQL môžete získať presne tie dáta, ktoré potrebujete. Dopyty musia obsahovať polia, o ktoré máte záujem, a vzťahy medzi nimi. Napríklad:
```
```suggestion
## Dopytovanie dát

Vďaka GraphQL môžete získať presne tie dáta, ktoré potrebujete. Dopyty musia obsahovať polia, o ktoré máte záujem, a vzťahy medzi nimi. Napríklad:
```

danielschuster-muc commented on docs/pages/apis/graphql.sk.mdx:
````diff
@@ -0,0 +1,47 @@
+```
+V tomto dopyte chceme získať názov (`title`) a dátum vydania (`releaseDate`) prvých troch kníh, ako aj názov (`title`) a slug (`slug`) každej kapitoly vo všetkých troch knihách. Okrem toho chceme získať aj meno (`name`) postavy so slugom `harry-potter`.
+
+## Schéma
+Kompletnú schému GraphQL API nájdete [tu](https://github.com/danielschuster-muc/potter-db/blob/master/backend/app/graphql/schema.json).
````
```suggestion
## Schéma

Kompletnú schému GraphQL API nájdete [tu](https://github.com/danielschuster-muc/potter-db/blob/master/backend/app/graphql/schema.json).
```

danielschuster-muc commented on docs/pages/apis/rest.sk.mdx:
```diff
@@ -0,0 +1,158 @@
+Na uľahčenie integrácie poskytujeme špecifikáciu v súlade s [OpenAPI](https://openapis.org) štandardom, ktorá popisuje jednotlivé koncové body API, dátové modely a schémy požiadaviek a odpovedí. Naše API sa riadi verziou 3.0.3 OpenAPI štandardu.
+Aktuálnu API špecifikáciu nájdete [tu](https://api.potterdb.com/v1/openapi.json).
+
+## Formát JSON:API
```
```suggestion
## Formát JSON:API

```

danielschuster-muc commented on docs/pages/getting-started.sk.mdx:
```diff
@@ -0,0 +1,42 @@
+import { Bleed } from 'nextra-theme-docs'
+
+# REST vs. GraphQL
```
```suggestion
# REST vs. GraphQL

```

danielschuster-muc commented on docs/pages/getting-started.sk.mdx:
```diff
@@ -0,0 +1,42 @@
+</table>
+}
+
+## Obmedzenie maximálneho počtu požiadaviek
```
```suggestion
## Obmedzenie maximálneho počtu požiadaviek

```

danielschuster-muc commented on docs/pages/resources/potions.sk.mdx:
```diff
@@ -0,0 +1,99 @@
+# Odvary (potions)
```
```suggestion
# Odvary (potions)

```

danielschuster-muc commented on docs/pages/resources/spells.sk.mdx:
```diff
@@ -0,0 +1,87 @@
+# Zaklínadlá (spells)
```
```suggestion
# Zaklínadlá (spells)

```

danielschuster-muc reviewed (changes requested):
Thanks for your contribution! Looks pretty good so far. Just some smaller things, see my comments.

danielschuster-muc commented:
Hi @romana-la, are there any updates from your side? Could you please resolve the requested changed, so I can merge you pr?

romana-la commented:
Hi @danielschuster-muc, thanks for your feedback and sorry that it took me so long to get to it. I have implemented most of your suggestions. Re. having resources in both English and Slovak - I have dropped English from the statistics table but I would still keep it in the table of contents and titles. When somebody makes a request, they will have to specify the resource in the request URL in English. When they then look in the documentation for the explanation of attributes to interpret the response, they will be likely looking for the English word.

danielschuster-muc commented:
> Hi @danielschuster-muc, thanks for your feedback and sorry that it took me so long to get to it. I have implemented most of your suggestions. Re. having resources in both English and Slovak - I have dropped English from the statistics table but I would still keep it in the table of contents and titles. When somebody makes a request, they will have to specify the resource in the request URL in English. When they then look in the documentation for the explanation of attributes to interpret the response, they will be likely looking for the English word.

Yes, you definitely have a point, makes sense to have the resources names in English and Slovak. Thank you for your input.

danielschuster-muc reviewed (approved):
Looks very nice! Thanks for your contribution.

kolja-lucht commented:
Hey @danielschuster-muc 
could provide the German translation 🧙‍♀️

danielschuster-muc commented:
> Hey @danielschuster-muc could provide the German translation 🧙‍♀️

sure! maybe you can leave a short note in #738

kolja-lucht commented:
Oops, I thought I had commented on that issue. Didn't realize I was in a PR 😅 Sorry about that @danielschuster-muc
